packages/back-end/src/controllers/metrics.ts

```diff
@@ -91,15 +91,9 @@ export async function deleteMetric(
     return;
   }
 
-  req.checkPermissions(
-    "createAnalyses",
-    metric?.projects?.length ? metric.projects : ""
-  );
```

Comment:
Is removing this intentional?

Reply:
It was - it wasn't clear to me why a user would need the `createAnalyses` permission in order to remove a metric.

If we do want this to be a requirement, I think it would be best to add it to the `canDeleteMetric` function so we don't have to do it twice.

Is this ultimately trying to protect from a user removing a metric that is associated with an experiment that they don't have permission to impact? If so, it feels like it would be better to check that against all of the experiments, since a metric could be in all projects, and be associated with an experiment that is in a project hidden to a user, which the check above wouldn't catch.

Reply:
Not saying your change is wrong, just flagging anything in my review that looked out of the ordinary.

I think this was there because we're doing `removeMetricFromExperiments` after we delete, but I agree it's probably fine to remove this check.  It's a super limited form of editing the experiment, so you shouldn't need full access in order to do that.